test/integration/testDataType.js

```diff
@@ -244,6 +245,34 @@ describe('Test DataType', function ()
       );
     });
 
+    it('testVariantXML', function (done)
+    {
+      async.series(
+        [
+          function (callback)
+          {
+            testUtil.executeCmd(connection, createTableWithVariant, callback);
```

Comment:
can we create describe for variant and in before/afterEach/after add createTable/truncateTable/dropTable?

Reply:
Sure, I also copied the way you used `executeCmdAsync` from the large result test case